snippets/computed.cson

```diff
@@ -0,0 +1,146 @@
+'.source.coffee':
+
+
+
+
+  'Ember Snippets: computed properties':
+    'prefix': 'computed--help'
+    'body':
+      """
+      ${1:
```

Comment:
They are written to be used with the knowledge we have already in mind.
But however we as human nonetheless Sometimes Forget.
these help files are just for a quick recap for when we forget or suspect.
Not to be used each time.